README.md

```diff
@@ -17,3 +17,39 @@ http://godoc.org/github.com/fhs/go-netcdf/netcdf
 How to install:
 
 	$ go get github.com/fhs/go-netcdf/netcdf
+
+## Example
```

Comment:
No need to have examples here. These examples don't even compile (see below). You can add examples so that they get run by 'go test' and godoc presents them next to the documentation. See https://blog.golang.org/examples